compiler/rustc_metadata/src/creader.rs

```diff
@@ -746,15 +750,13 @@ impl<'a> CrateLoader<'a> {
         // Sanity check the loaded crate to ensure it is indeed a panic runtime
         // and the panic strategy is indeed what we thought it was.
         if !data.is_panic_runtime() {
-            self.sess.err(&format!("the crate `{}` is not a panic runtime", name));
+            self.sess.emit_err(CrateNotPanicRuntime { crate_name: name.to_string() });
```

Comment:
You can probably just use a `Symbol` field in the struct.

Reply:
✅